src/src/lms/lms.service.ts

```diff
@@ -419,5 +379,428 @@ export class LMSService {
 		}
 	}
 
-	//delete
+	//cron issue certificate run every 5 minutes
+	@Cron(CronExpression.EVERY_5_MINUTES)
```

Comment:
can we remove cron in folder called `crons`?

Reply:
remove?

Reply:
@manojLondhe If you are saying the move to crons folder, if I need to use only lms module in another project or Shiksha it will be without cron job

Reply:
i moved this in crons folder